apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -93,6 +105,34 @@ class ResultDetail extends React.Component<Props, State> {
             onChangeCountryCode={this.handleChangePhoneCountryCode}
           />
         </ScrollView>
+        <PriceSummary
```

Comment:
what about the React compound pattern? It will be more readable I think but it is just an idea. 
```
<PriceSummary>
<PriceSummary.Visible>
	subtotal
<PriceSummary.Visible>
<PriceSummary.CollapseContent>
	<TableRow label="1x Passenger" value="123 Kč" />
    <TableRow label="1x Cabin bag" value="123 Kč" />
    <TableRow label="1x Personal item" value="123 Kč" />subtotal
</PriceSummary.CollapseContent>
</PriceSummary>
```
What do you think?
https://blog.logrocket.com/guide-to-react-compound-components-9c4b3eb482e9

Reply:
I'm not sure about that, because 
- you can't ensure that there will be only one component of this type (`PriceSummary.Visible`, `<PriceSummary.CollapseContent>`)
- Also component should respect position where component is rendered in children. But we can't do it. What if `PriceSummary.CollapseContent` and `PriceSummary.Visible` will be inverted? I think behaviour should be the same, but I'm not sure if this design is good for us in that case.

Reply:
OK it was just an idea, so keep it as it is 😄